### ISSUE TITLE
Add Sub-factions

### ISSUE THREAD

**Issue by ebruning** [AppSynergy/shadow-war-armageddon-roster#11]
With Orks, Guard and Space Marines the sub-factions determine the skills. A quick look and it seems like this only applies to these teams.

For guard, there are Cadians, Catachans and Steel Legion. For Cadians only the leader can have Combat but for Catachans and Steel Legion everyone can have Combat

**Comment by AppSynergy** [comment]
I didn't know there were sub-factions for these, interesting.

Does it change anything apart from the skill sets?

**Comment by ebruning** [comment]
Nope not at all, only Space Wolves (see my comment in #12)  It didn't look like skills were implemented yet (or I hadn't noticed) so I just left the text in.

It might just be easier to add Space Wolves as a Kill Team and just remove  Novitiate Scouts from the list. It would be the exact same file as Space Marines (not ideal but I don't think weapons and costs will change). If that makes sense I can do it later today

**Comment by AppSynergy** [comment]
Yeah, I agree. I've just pasted together what I think the Space Wolves list looks like (add 3 special weapons, rm novitiate), but I don't have the book in front of me, think that's right though.

I'll leave this open to remind me the skills may be different. Currently nothing is done with skills, but I thought I might as well add them in while I was trawling through the pdf!

**Comment by ebruning** [comment]
I took a look at the TOML and it looks good

**Comment by AppSynergy** [comment]
Sweet, it's live!

**Comment by ebruning** [comment]
Since skills are really only used after a game and the website builds kill teams but doesn't track them. I think it's ok for now. BTW Looks Good

I just had a thought, and it's my fault, the official title is Space Wolves Special Weapons, I'll send a PRE in a sec,

**Comment by AppSynergy** [comment]
> builds kill teams but doesn't track them

Yet! :grinning:

**Comment by ebruning** [comment]
Sweet!
